### PR TITLE
DOC-3569: Release Notes week ending Feb 3

### PR DESCRIPTION
**Pull request by catong**
## [DOC-3569](https://openedx.atlassian.net/browse/DOC-3569)
Release notes for week ending Feb 3

### Date Needed

Monday, Feb 6

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @katymyw for Insights
- [ ] Subject matter expert: @cahrens for TNL-6357
- [x] Subject matter expert: @alisan617 or @bjacobel for TNL-6363 and TNL-6364
- [x] Doc team review: @grantgoodmanedX 
- [x] Product review: @marcotuts @sstack22 

FYI: @jaakana 

### HTML Draft Output for Reviewers
- [x] http://draft-release-notes.readthedocs.io/en/latest/2017/2017-02-03.html

### Testing
- [x] Ran ./run_tests.sh without warnings or errors

### Post-review

- [x] Squash commits

### CODE REVIEW

**Review by bjacobel — APPROVED** [review]

**Comment by grantgoodmanedX** [comment]
The draft release note looks good to me. +1

**Comment by marcotuts** [comment]
👍

**Comment by catong** [comment]
@katymyw has given OK via Hipchat

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1407

File: en_us/release_notes/source/2017/lms/lms_2017-02-03.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+When the **Show Answer** setting is active for a problem, a correct answer is
```
Comment (cahrens) [inline en_us/release_notes/source/2017/lms/lms_2017-02-03.rst:1]: @catong 
This fix was only for multiple choice and checkbox problems. Other types of problems show their solutions in other ways (for instance, a problem with an input actually shows the numerical or text answer).

How about "When the Show Answer settings is active for a Multiple Choice or Checkboxes problem, a correct..."

Comment (catong) [reply en_us/release_notes/source/2017/lms/lms_2017-02-03.rst:1]: Thanks @cahrens; addressed in https://github.com/edx/edx-documentation/pull/1410

**Pull request merged**